data/transition-sites/adjudicatorsoffice.yml

```
@@ -0,0 +1,6 @@
+---
+site: adjudicatorsoffice
+whitehall_slug: the-adjudicator-s-office
+homepage: https://www.gov.uk/government/organisations/the-adjudicator-s-office
+tna_timestamp: 20190501125725
+host: www.adjudicatorsoffice.gov.uk
```

```suggestion
host: www.adjudicatorsoffice.gov.uk
aliases:
  - adjudicatorsoffice.gov.uk
```

The non-www version counts as a different host so it's usually good to include it here so it continues to work when it's transitioned.